apps/web/lib/routing/getStaticProps.ts

```diff
@@ -0,0 +1,89 @@
+import fs from "fs";
+import matter from "gray-matter";
+import path from "path";
+import { z } from "zod";
+
+import { getAppWithMetadata } from "@calcom/app-store/_appRegistry";
+import { getAppAssetFullPath } from "@calcom/app-store/getAppAssetFullPath";
+import { IS_PRODUCTION } from "@calcom/lib/constants";
+import prisma from "@calcom/prisma";
+
+export const sourceSchema = z.object({
+  content: z.string(),
+  data: z.object({
+    description: z.string().optional(),
+    items: z
+      .array(
+        z.union([
+          z.string(),
+          z.object({
+            iframe: z.object({ src: z.string() }),
+          }),
+        ])
+      )
+      .optional(),
+  }),
+});
+
+export const getStaticProps = async () => {
+  const appName = "routing-forms";
+
+  const appMeta = await getAppWithMetadata({
+    slug: appName,
+  });
+
+  const appFromDb = await prisma.app.findUnique({
+    where: { slug: appName },
+  });
+
+  const isAppAvailableInFileSystem = appMeta;
+  const isAppDisabled = isAppAvailableInFileSystem && (!appFromDb || !appFromDb.enabled);
+
+  if (!IS_PRODUCTION && isAppDisabled) {
+    return {
+      props: {
+        isAppDisabled: true as const,
+        data: {
+          ...appMeta,
+        },
+      },
+    };
+  }
+
+  if (!appFromDb || !appMeta || isAppDisabled) return { notFound: true } as const;
+
+  const isTemplate = appMeta.isTemplate;
+  const appDirname = path.join(isTemplate ? "templates" : "", appFromDb.dirName);
+  const README_PATH = path.join(process.cwd(), "..", "..", `packages/app-store/${appDirname}/DESCRIPTION.md`);
```

Comment:
Hard-coded paths could cause issues in different environments, also there should be path validation.

Reply:
Do you have any idea of how can we avoid hard coded Path, I can't think of it ??

Reply:
we can probably store all path configurations in a centralized config file, and use that file